presto-native-execution/presto_cpp/main/operators/BroadcastWrite.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "presto_cpp/main/operators/FileBroadcast.h"
+#include "velox/core/PlanNode.h"
+#include "velox/exec/Operator.h"
+
+namespace facebook::presto::operators {
+
+class BroadcastWriteNode : public velox::core::PlanNode {
+ public:
+  BroadcastWriteNode(
+      const velox::core::PlanNodeId& id,
+      const std::string& broadcastWriteBasePath,
```

Comment:
broadcastWriteBasePath -> basePath since 'broadcastWrite' is already part of the class name; ditto member variable name and getter